plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/ModelLayerWrapper.kt

```diff
@@ -17,6 +17,7 @@ internal class ModelLayerWrapper(
     layerProperties["model-scale"] = Value(modelScale.map(::Value))
     layerProperties["model-rotation"] = Value(modelRotation.map(::Value))
     layerProperties["model-translation"] = Value(modelTranslation.map(::Value))
+    layerProperties["model-opacity"] = Value(1.0)
```

Comment:
all other properties are passed from above and here it's pure `1.0`... Perhaps pass it thru ctor as well?